### PR TITLE
[15.0][MIG] web_ir_actions_act_view_reload

### PR DESCRIPTION
**Pull request by ryanc-me**
Fortunately, Odoo 15.0 is still using the old-style controllers/renderers, so this wasn't too difficult once we can access the legacy `widget`.

The 16.0 migration for this module might be interesting!

### CODE REVIEW

**Comment by ryanc-me** [comment]
Note, previous attempt at #2053

**Comment by ryanc-me** [comment]
Note, previous attempt at #2240

**Review by pedrobaeza — DISMISSED** [review]
Thanks for the contribution.

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

**Comment by ryanc-me** [comment]
Thanks Pedro.

I see that the fork branch should be named `15.0-mig-$MODULE` - I can't change that without creating a new PR. Will the current state work for you?

**Comment by pedrobaeza** [comment]
Don't worry about the branch name in this case, as such name is only a transient one that is not transferred to any place once the pull request is merged. It's better to not use `15.0` one, as you can have several pull requests for the same branch, though, but for following ones ( that I hope to be a lot :smile: )

**Comment by ryanc-me** [comment]
Thanks @pedrobaeza, for sure there will be some more PRs to come!

**Review by etobella — CHANGES_REQUESTED** [review]
Something does not work:

I followed this steps:

- Access runboat
- Create a server action on any model with the code `action={'type': 'ir.actions.act_view_reload'}`
- Create the Contextual Action
- Refresh
- Access the model and press the button
- The following error is raised:

```
Error: The ActionManager service can't handle actions of type ir.actions.act_view_reload
    at doAction (http://oca-web-15-0-pr2272-3d7bde7f5331.runboat.odoo-community.org/web/assets/debug/web.assets_backend.js:25001:23) (/web/static/src/webclient/actions/action_service.js:1138)
```

File: web_ir_actions_act_view_reload/__manifest__.py
Comment (wildi1) [inline web_ir_actions_act_view_reload/__manifest__.py]: In my case the js was not loaded this way. Changing line 18 to the following worked for me:
`"web_ir_actions_act_view_reload/static/src/**/*.esm.js",`

Comment (ryanc-me) [reply web_ir_actions_act_view_reload/__manifest__.py]: @wildi1 Thanks - good catch. I guess the JS was cached in my browser during tests. Fixed now 👍

Comment (wildi1) [reply web_ir_actions_act_view_reload/__manifest__.py]: @ryanc-me :+1:

**Review by CarlosRoca13 — APPROVED** [review]
Tested @etobella use case and tested if is added the action by code.

It seems to be working fine

Thanks!

**Comment by ryanc-me** [comment]
Thanks @CarlosRoca13!

Ping @etobella, it looks like the merge is blocked because of a change request. Could you please take a look?

**Comment by StefanRijnhart** [comment]
@etobella I followed your scenario and I did not encounter this problem. Running the action correctly triggered the method that this module registers for the action type.

image

**Review by StefanRijnhart — APPROVED** [review]

**Review by etobella — APPROVED** [review]
Thanks!!! Now it works perfectly
:smile: 

/ocabot merge nobump

**Pull request merged**
